modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/AuthorizationRepository.java

```diff
@@ -102,4 +103,26 @@
      * @return user permissions
      */
     UserPermissions checkSuperAdminPermissions(UserInfo.Username userID, Application.Name applicationName);
+
+    /**
+     * Assign user to superadmin role
```

Comment:
minor. to be consistent with the comment. suggest to change it to "Assign candidate user to..."